server_admin/topics/sessions/revocation.adoc

```diff
@@ -4,9 +4,14 @@
 === Revocation policies
 [role="_abstract"]
 
+ifeval::[{project_community}==true]
+If your system is compromised, you can revoke all active sessions and access tokens by clicking the `Sessions` screen's *Revocation* button in the 'Action' menu.
+endif::[]
+ifeval::[{project_product}==true]
 If your system is compromised, you can revoke all active sessions and access tokens by clicking the `Sessions` screen *Revocation* tab.
+endif::[]
 
 .Revocation
-image:{project_images}/revocation.png[]
+image:{project_images}/revocation.png[A screenshot showing the 'Revocation' functionality.]
```

Comment:
If your system is compromised, you can revoke all active sessions and access tokens.

.Procedure
ifeval::[{project_community}==true]
. Click *Sessions* in the menu.
. From the *Actions* list, select *Sign out all active sessions*.
endif::[]
ifeval::[{project_product}==true]
+
endif::[]

Reply:
Update copy 5